main/src/ui/chat_input/smiley_converter.vala

```diff
@@ -25,6 +27,13 @@ class SmileyConverter {
         smiley_translations[":("] = "🙁";
         smiley_translations[":'("] = "😢";
         smiley_translations[":/"] = "😕";
+        smiley_translations["<3"] = "🖤";
+        smiley_translations[":heart:"] = "🖤";
+        smiley_translations[":kiss:"] = "💋";
+        smiley_translations[":-*"] = "💋";
+        smiley_translations[":*"] = "💋";
+        smiley_translations[":thumbsup:"] = "👍";
+        smiley_translations["+1"] = "👍";
```

Comment:
I would not do this, as it might corrupt calculations or even links. Try (y) and (n) for example.